app/helper/rtForm.php

```diff
@@ -1075,6 +1080,130 @@ public function get_select( $attributes = '' ) {
 		public function display_select( $args = '' ) {
 			echo $this->get_select( $args );
 		}
+
+		/**
+		 * Generate rtmedia html button in admin options.
+		 *
+		 * @access protected
+		 *
+		 * @param $attributes
+		 *
+		 * @return string
+		 * @throws rtFormInvalidArgumentsException
+		 */
+		protected function generate_button( $attributes ) {
+
+			$element = 'rtButton';
+			if ( is_array( $attributes ) ) {
```

Comment:
Avoid large `if` block. Throw an exception if `$attributes` is not an array. else further code will execute. So you don't need `else` condition.